catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/AttributeValueNormalizer.java

```diff
@@ -113,6 +116,8 @@ public String normalizeForXml(String property, String value) {
       return "";
     }
 
+    //TODO look into exactly how relative gets created - that or find out if its okay to store the between
```

Comment:
Address or create an issue.

Reply:
Didn't meant to include these changes. Gonna tear this down and put up a new one.

Reply:
Big whoopsie. This was my bug the whole time.
![image](https://user-images.githubusercontent.com/51682089/66515462-061af480-ea94-11e9-8e63-31065c24dc91.png)